.claude/skills/design/SKILL.md

```
@@ -0,0 +1,194 @@
+HashRouter behavior unchanged. Route names intact:
+```
+/#/projects
+/#/tasks
+```
+
+### CSS scope
+All styles under `#wedevs-project-manager`. CSS vars on `:root`.
+
+### Class naming
+Every custom class prefixed `pm-`:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)